### PR TITLE
Update naming schemes in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by projectoriented**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by projectoriented** [comment]
It seems like the `.CONTRIBUTING.md` file cannot be edited. I'll close this for now and consult the nf-core team tomorrow.

File: .github/CONTRIBUTING.md
Comment (jemten) [inline .github/CONTRIBUTING.md:107]:
```suggestion
Default options for modules should be defined in the [conf/modules.config](../conf/modules.config). If you have a module that you would like to reuse in a subworkflow with non-default options, we recommend modifying the defaults within the context of the subworkflow. For a working example, please have a look at how the default options for bcftools norm tool are modified in both [subworkflows/local/deepvariant_caller.nf](../subworkflows/local/deepvariant_caller.nf) and [subworkflows/local/prepare_vcf.nf](../subworkflows/local/prepare_vcf.nf)
```
The last link wont work until we merge your PR @ramprasadn :)

**Review by jemten — APPROVED** [review]
Looks good to me but would be nice with some more feedback

**Comment by projectoriented** [comment]
@ramprasadn We're going ahead and merging this PR. If you'd like to make further edits, feel free to make another PR 😇

**Pull request merged**
